automated_archive/aa.py

```diff
@@ -164,26 +191,17 @@ def _create_mysql(args, FILES, log):
             FILES[i].get("Summary", "").replace("'", "\\'"),
             _extract_tags(args, FILES[i]),
             _extract_characters(args, FILES[i]),
-            datetime.datetime.strptime(
-                FILES[i].get(
-                    "PrintTime",
-                    FILES[i].get(
-                        "DatePrint",
-                        FILES[i].get(
-                            "Date", str(datetime.datetime.now().strftime("%m/%d/%y"))
-                        ),
-                    ),
-                ),
-                "%m/%d/%y",
-            ).strftime("%Y-%m-%d"),
+            _extract_date(args, FILES[i], log),
             FILES[i].get("Location", "").replace("'", "\\'"),
             FILES[i]
             .get("LocationURL", FILES[i].get("StoryURL", ""))
```

Comment:
I wonder why these lines weren't indented, or is that a Python thing I've forgotten?

Reply:
All the items in this array have the same indentation, though it is kinda confusing that the ruff formatter broke up certain lines but not others (like the Location line vs the LocationURL line). So it may look odd but it is fine, unless you're talking about something else?

Reply:
Just about it looking odd and more difficult for a human to interpret since continuation lines are usually indented relative to their first line. However, this file isn't a high priority so if ruff is going to be run on the whole repo, we'll have to let it do its thing to avoid noisy diffs!